cond.go

```
@@ -186,7 +186,7 @@ func (c *Cond) LTE(field string, value interface{}) string {
 
 // In is used to construct the expression "field IN (value...)".
 func (c *Cond) In(field string, values ...interface{}) string {
-	if len(field) == 0 {
+	if len(field) == 0 || len(values) == 0 {
```

I'm wondering if this is what users will intuitively expect. I wouldn't. If I have an "allowlist" of values that I'm passing here, and it's empty for some reason, that query should yield no results, not silently ignore the filter and thus actually widen the result set.

---

Good suggestion. Accepted.